Localization/Tools/ru-RU_Mods.DragonLens.Tools.hjson

```diff
@@ -653,12 +653,12 @@ Hitboxes: {
 	Description: Отображает хитбоксы
 
 	Options: {
-		NPC: Хитбоксы существ
-		Projectile: Хитбоксы снарядов
-		Player: Хитбоксы игроков
-		Item: Хитбоксы предметов
-		// Melee: Melee Hitboxes
-		// TileEntity: Tile Entity Hitboxes

```

Comment:
I merged this anyway, but in English we keep "Hitboxes"; is there a reason you remove them here?